src/adapter/src/catalog/builtin.rs

```diff
@@ -1699,7 +1699,8 @@ pub static MZ_SECRETS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
         .with_column(
             "privileges",
             ScalarType::Array(Box::new(ScalarType::MzAclItem)).nullable(false),
-        ),
+        )
+        .with_column("oid", ScalarType::Oid.nullable(false)),
```

Comment:
It's a bit unfortunate that we need to add the column to the end so it doesn't match the other tables, but adding the column in the middle would be a breaking change.

Reply:
What does it break? Things that rely on `*` having a specific order?

Reply:
Yeah, `*` having a specific order is one issue. Another is you can partially alias the columns of a table and that depends on the column order. For example:
```
postgres=# CREATE TABLE t (a INT, b INT, c INT);
CREATE TABLE
postgres=# SELECT a1, b1, c FROM t AS t1 (a1, b1);
 a1 | b1 | c 
----+----+---
(0 rows)

```

Reply:
I'm fine merging as is. I'm also fine making this consistent with the other tables because I don't think many people are querying mz_secrets, and we can publish this as a breaking change in the release notes. I'd prefer doing the breaking change while we don't have many users.

Reply:
I personally don't think it's worth combing through all the prod environments to check if this is will break anyone's environment, but if you or someone else want to volunteer to do that check then I also think it would look cleaner having the columns match in order.

Reply:
Or if you think that we don't need to check beforehand that works too.

Reply:
You can ask in the field eng channel if anyone is using this. We've safely made breaking changes before when they confirm no one is using a certain thing.

Reply:
I'm going to merge this as is since it's blocking another PR of mine, and I'll open a second one with the breaking change.

Reply:
https://github.com/MaterializeInc/materialize/pull/19536